hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/ReplicateUpdateToCallerOperation.java

```diff
@@ -33,7 +34,8 @@
 /**
  * This operation will update the local record store with the update received from local/remote partition owner.
  */
-public class ReplicateUpdateToCallerOperation extends AbstractSerializableOperation implements PartitionAwareOperation {
+public class ReplicateUpdateToCallerOperation extends AbstractSerializableOperation
```

Comment:
Same as for `ReplicateUpdateOperation`, this is a result of a successful run of a `put` or `remove`. I would allow it not to cause inconsistencies.

Reply:
Now that I think about it, what's the point of rejecting backup operations such as `AcquireBackupOperation`? This means that the original operation has finished but the quorum was lost before the backup was applied. So we just get that the primary replica diverges from the backup replica. WDYT?

Reply:
It needs a broader discussion. It's not sure whether we are running the backup operation on the correct node, because the backup partition owner might be in the other brain so I don't see any win-win here, it's not a big harm either. I'm adding this on TODO here #11876 If we agree on this I need to fix it in all structures anyway.